contracts/MultiSend.sol

```diff
@@ -1,42 +1,55 @@
 pragma solidity ^0.4.18;
 
 import "giveth-common-contracts/contracts/Escapable.sol";
+import "giveth-common-contracts/contracts/SafeMath.sol";
 
 // TightlyPacked is cheaper if you need to store input data and if amount is less than 12 bytes.
 // Normal is cheaper if you don't need to store input data or if amounts are greater than 12 bytes.
 // Supports deterministic deployment. As explained here: https://github.com/ethereum/EIPs/issues/777#issuecomment-356103528
 contract MultiSend is Escapable {
   
   address CALLER = 0x839395e20bbB182fa440d08F850E6c7A8f6F0780;
-  address DESTINATION = 0x8ff920020c8ad673661c8117f2855c384758c572;
+  address DESTINATION = 0x8Ff920020c8AD673661c8117f2855C384758C572;
 
   function MultiSend() Escapable(CALLER, DESTINATION) public {}
   
   function multiTransferTightlyPacked(bytes32[] _addressAndAmount) payable public returns(bool) {
+    uint toReturn = msg.value;
     for (uint i = 0; i < _addressAndAmount.length; i++) {
         _safeTransfer(address(_addressAndAmount[i] >> 96), uint(uint96(_addressAndAmount[i])));
+        toReturn = SafeMath.sub(toReturn, uint(uint96(_addressAndAmount[i])));
```

Comment:
since `uint(uint96(_addressAndAmount[i])` is used twice it would probably be good to save it to a variable.